pkg/apis/multiclusterdns/v1alpha1/multiclusterservicednsrecord_types.go

```diff
@@ -27,6 +27,8 @@ type MultiClusterServiceDNSRecordSpec struct {
 	FederationName string `json:"federationName,omitempty"`
 	// DNSSuffix is the suffix (domain) to append to DNS names
 	DNSSuffix string `json:"dnsSuffix,omitempty"`
+	// RecordTTL is the TTL for DNS records created for this Service, if omitted a default would be used
```

Comment:
Can we add the unit of time measurement for this field either as a comment or as part of the field name itself e.g. `RecordTTLSeconds`?

Also, I just wanted to point out that we use the `TTL` type in the `EndPoint` API but not here. I don't necessarily have a preference either way.

Reply:
Have updated the comment to specify the unit is in Seconds. Also updated the type to TTL to be in sync.